src/connectors/connectStateResults.js

```diff
@@ -0,0 +1,45 @@
+const connectStateResults = (renderFn, unmountFn) => (widgetParams = {}) => ({
```

Comment:
There's not tests for this connector. It would be for asserting `isFirstRender` values and calling `unmountFn`. Is that on purpose?